test/test_allocations.jl

```diff
@@ -11,28 +11,32 @@ end
   @testset "allocs QPSData" begin
     for problem in qp_problems_Matrix
       nlp_qps = eval(Symbol(problem * "_QPSData"))()
-      test_only_zeros(test_allocs_nlpmodels(nlp_qps))
+      allocs = test_allocs_nlpmodels(nlp_qps; linear_api=true, exclude=[jac_op])
```

Comment:
Did you create an issue connected to this? It doesn't seem normal.

Reply:
No I didn't opened an issue. It seems to appear around Julia 1.11.4 / 1.11.5.
I got a similar issue with Krylov.jl.

Reply:
So, it is a LinearOperators problem?

Reply:
It could be. I also suspect that `FastClosures.jl`.

Reply:
Could you create issues so we can track this? Thanks!